plugins/woocommerce/templates/checkout/form-pay.php

```diff
@@ -67,6 +67,8 @@
 		</tfoot>
 	</table>
 
+	<?php do_action( 'woocommerce_pay_order_before_payment' ); ?>
```

Comment:
I wonder if the PHP linting action is broken? [It seems to have run](https://github.com/woocommerce/woocommerce/actions/runs/5968747311/job/16193214078?pr=37588), but there is none of the usual output I'd expect. Anyway ... we have a `WooCommerce.Commenting.CommentHooks.MissingHookComment` rule that we should follow here. How about:

```suggestion
	<?php
	/**
	 * Triggered from within the checkout/form-pay.php template, immediately before the payment section.
	 *
	 * @since 8.2.0
	 */
	do_action( 'woocommerce_pay_order_before_payment' ); 
	?>
```